### PR TITLE
Schema-related changes

### PR DESCRIPTION
**Pull request by mpinter**
This, along with the changes made in schema.json and uiSchema.json on staging env, should resolve the following:
- https://github.com/bratislava/konto.bratislava.sk/issues/435
- https://github.com/bratislava/konto.bratislava.sk/issues/491
- https://github.com/bratislava/private-konto.bratislava.sk/issues/19 (only a change in schema was needed)

### CODE REVIEW

File: next/components/forms/widget-components/Upload/UploadFileCard.tsx [hunk 1]
```diff
@@ -76,15 +76,15 @@ const UploadFileCard = ({
             <div className="flex grow flex-col">
               <h3 className="break-words font-bold text-gray-800">{fileInfo.fileName}</h3>
               <div className="flex gap-2">
-                {/* TODO: Improve */}
-                {fileInfo.canDownload && (
+                {/* TODO: Temp fix - without this nothing can be clicked on after uploading a file - tracked in #499 */}
+                {/* {fileInfo.canDownload && (
                   <ReactAriaButton
```
Comment (MarekBodingerBA) [inline next/components/forms/widget-components/Upload/UploadFileCard.tsx:81]: Just remove "after:absolute after:inset-0".

File: next/components/forms/FormPage.tsx [hunk 2]
```diff
@@ -34,7 +34,8 @@ const FormPage = () => {
 
   return (
     <>
-      <FormHeader title={schema.title} />
+      {/* Root level ui:options can be used for custom props */}
+      <FormHeader uiSchemaOptions={uiSchema['ui:options']} title={schema.title} />
```
Comment (MarekBodingerBA) [inline next/components/forms/FormPage.tsx:38]: `useFormState` can read all of this. There's no need for new props.

File: next/components/forms/widget-wrappers/UploadWidgetRJSF.tsx [hunk 3]
```diff
@@ -93,6 +94,7 @@ const UploadWidgetRJSF = (props: UploadWidgetRJSFProps) => {
         sizeLimit={size}
         supportedFormats={supportedFormats}
         disabled={disabled || readonly}
+        additionalLinks={additionalLinks}
```
Comment (MarekBodingerBA) [inline next/components/forms/widget-wrappers/UploadWidgetRJSF.tsx:97]: Maybe WidgetWrapper is more suitable even?

**Review by MarekBodingerBA — APPROVED** [review]

File: next/components/forms/widget-wrappers/UploadWidgetRJSF.tsx (hunk 3 above)
Comment (mpinter) [reply next/components/forms/widget-wrappers/UploadWidgetRJSF.tsx]: you are right - moved there and added to every widget-wrapped element

**Pull request merged**